x-pack/plugins/reporting/server/export_types/csv_saved_object/lib/get_sharing_data.test.ts

```diff
@@ -151,6 +151,49 @@ describe('get_sharing_data', () => {
     ]);
   });
 
+  it('with saved search containing a filter', async () => {
```

Comment:
This doesn't test anything new: it was added to make sure that filters stored on the saved search can be a singular object, or an array.